### PR TITLE
[BUG]fix add sample notebooks

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description
Add sample notebooks was failing in NEO as the sample data did not have a cluster to point to, so have updated the Apis to reflect the MDS id for samples

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/notebooks/components/main.tsx [hunk 1]
```diff
@@ -95,19 +95,32 @@ export class Main extends React.Component<MainProps, MainState> {
...
+      // If `MDS` is enabled, only fetch from the first endpoint.
+      return this.props.http
+        .get(`${NOTEBOOKS_API_PREFIX}/savedNotebook/`)
+        .then((savedNotebooksResponse) => {
+          this.setState({ data: savedNotebooksResponse.data });
+        })
+        .catch((err) => {
+          console.error('Issue in fetching the notebooks', err.body.message);
+        });
+    } else {
```
Comment (joshuali925) [inline public/components/notebooks/components/main.tsx:108]: nit. else is not necessary

File: public/components/notebooks/components/main.tsx [hunk 2]
```diff
@@ -95,19 +95,32 @@ export class Main extends React.Component<MainProps, MainState> {
...
+      // If `MDS` is not enabled, fetch from both endpoints and combine the data.
+      return Promise.all([
+        this.props.http.get(`${NOTEBOOKS_API_PREFIX}/savedNotebook/`),
+        this.props.http.get(`${NOTEBOOKS_API_PREFIX}/`),
```
Comment (joshuali925) [inline public/components/notebooks/components/main.tsx:112]: it's not very clear what the difference is, maybe add a comment to say `/savedNotebook/` API returns notebooks stored as saved objects, and the other one returns notebooks stored as observability objects?

File: public/components/notebooks/components/note_table.tsx [hunk 3]
```diff
@@ -157,11 +177,28 @@ export function NoteTable({
...
+    };
     setModalLayout(
-      getSampleNotebooksModal(closeModal, async () => {
-        closeModal();
-        await addSampleNotebooks();
-      })
+      getSampleNotebooksModal(
+        closeModal,
+        async () => {
+          console.log(selectedDataSourceId, selectedDataSourceLabel);
```
Comment (joshuali925) [inline public/components/notebooks/components/note_table.tsx:192]: remove console log

File: public/components/notebooks/components/note_table.tsx [hunk 4]
```diff
@@ -80,6 +98,8 @@ export function NoteTable({
   const [searchQuery, setSearchQuery] = useState('');
   const location = useLocation();
   const history = useHistory();
+  const [_dataSourceMDSId, setDataSourceMDSId] = useState<string | undefined>('');
+  const [_dataSourceMDSLabel, setDataSourceMDSLabel] = useState<string | undefined>('');
```
Comment (joshuali925) [inline public/components/notebooks/components/note_table.tsx:102]: unused?

Comment (sumukhswamy) [reply public/components/notebooks/components/note_table.tsx:98]: this I dont use the variable but I have used the set function

**Review by ps48 — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

**Pull request merged**